backend/src/akvo/lumen/lib/transformation/derive.clj

```diff
@@ -43,27 +43,32 @@
   or row[\"foo\"]. For every reference return a tuple with matched pattern and
   the row column as in [\"row.foo\" \"foo\"]."
   [code]
-  (let [re #"(?U)row.([\w\d]+)|row(?:\[')([^']*)(?:'\])|row(?:\[\")([^\"]*)(?:\"\])"
+  (let [re #"(?U)row.([\w\d]+)|row(?:\[')([^']*)(?:'\])(?:\[')([^']*)(?:'\])|row(?:\[')([^']*)(?:'\])|row(?:\[\")([^\"]*)(?:\"\])(?:\[\")([^\"]*)(?:\"\])|row(?:\[\")([^\"]*)(?:\"\])"
```

Comment:
I tried but didn't find a better way to do this regex

Reply:
Oh my :-)